src/force-app/main/default/classes/UserSelector.cls

```diff
@@ -0,0 +1,6 @@
+public with sharing class UserSelector {
+    public static Set<Id> getInactiveUserIds() {
+        Map<Id, User> inactiveUsers = new Map<Id, User>([SELECT Id FROM User WHERE isActive = false]);
```

Comment:
Why to run huge query to pull all inactive users if you may pull only users you're interested at?

15 mins to fix.